app/api.py

```diff
@@ -320,9 +360,16 @@ def get_flags_by_ticket(ticket_id: int):
         }
 
 
-# Update ticket status by id with enum : open, closed (soft delete)
 @api_v1_router.put("/tickets/{ticket_id}/status")
 def update_ticket_status(ticket_id: int, status: TicketStatus):
+    """Update the status of a ticket by ID.
+
+    This function is used to update the status of a ticket by its ID.
+
+    :param ticket_id: the ID of the ticket
```

Comment:
You don't need to provide `:param` for API routes, as this text will appear in swagger doc (and it's rather ugly, and unecessary).
You should however provide :param for other functions.